bundles/org.eclipse.jface/META-INF/p2.inf

```diff
@@ -0,0 +1,7 @@
+# Ensure that 4.36 is not installed with the 0.0.1,2024.4.0.202412101111 version of darkest dark devstyle.
+# https://github.com/eclipse-packaging/packages/issues/307
+requires.0.namespace=org.eclipse.equinox.p2.iu
+requires.0.name=com.genuitec.eclipse.theming.core.feature.feature.group
+requires.0.range = [0.0.1,2024.4.0.202412101111]
```

Comment:
```suggestion
requires.0.range = [0.0.1,9999)
```

As we don't know in what version it will be fixed or if it ever will be fixed.

Reply:
> Compromise: `[0.0.1,2025)` ?

Sounds reasonable to me, lets hope we don't need to increase this every year :-P

Reply:
> Maybe the number is not pretty enough? Not round enough? My patience for this is definitely not enough.

Exact :-)

Ed, we don't know what is the version policy there and whether a "breaking" change like in 2025.06 release would be fixed only in the next major bundle version. So having at least "not minor" version bump is something I would like to have to make sure we don't need to bump this version again here...

Reply:
I have to admit that I do not seem to understand the version constraints properly, as otherwise I would have also been in favor of using `[0.0.1,2024.4.1)` or `[0.0.1,2025.0.0)` (to force an "active version bump", not qualifier update or the like).
But, if I understand correct, since you are confident they will not work or they are not reasonable, I give +1 to the existing proposal. And of course, thank you very much for taking care of the change and being patient with all proposals made on it!